src/main/java/io/github/pmckeown/dependencytrack/finding/FindingsMojo.java

```diff
@@ -75,33 +83,46 @@ public class FindingsMojo extends AbstractDependencyTrackMojo {
     private FindingsPrinter findingsPrinter;
     private FindingsAnalyser findingsAnalyser;
     private FindingsReportGenerator findingsReportGenerator;
+    private PolicyAction policyAction;
+    private PolicyViolationsPrinter policyViolationsPrinter;
+    private PolicyAnalyser policyAnalyser;
 
     @Inject
     public FindingsMojo(ProjectAction projectAction, FindingsAction findingsAction, FindingsPrinter findingsPrinter,
                         FindingsAnalyser findingsAnalyser, FindingsReportGenerator findingsReportGenerator,
-                        CommonConfig commonConfig, Logger logger) {
+                        CommonConfig commonConfig, Logger logger, PolicyAction policyAction,
+                        PolicyViolationsPrinter policyViolationsPrinter, PolicyAnalyser policyAnalyser) {
```

Comment:
This triggered me to think that a new goal was needed.  There are too many parameters now to this constructor indicating this class is getting too big.